code/modules/client/preferences/entries/character/species_features/diona.dm

```diff
@@ -0,0 +1,356 @@
+/datum/preference/choiced/diona_leaves
+	db_key = "feature_diona_leaves"
+	preference_type = PREFERENCE_CHARACTER
+	category = PREFERENCE_CATEGORY_FEATURES
+	main_feature_name = "Diona Leaves"
+	should_generate_icons = TRUE
+	relevant_mutant_bodypart = "diona_leaves"
+
+/datum/preference/choiced/diona_leaves/init_possible_values()
+	var/list/values = list()
+
+	var/datum/universal_icon/diona_body = uni_icon('icons/effects/effects.dmi', "nothing")
+
+	var/list/body_parts = list(
+		BODY_ZONE_HEAD,
+		BODY_ZONE_CHEST,
+		BODY_ZONE_L_ARM,
+		BODY_ZONE_R_ARM,
+		BODY_ZONE_L_LEG,
+		BODY_ZONE_R_LEG
+	)
+
+	for (var/body_part in body_parts)
+		diona_body.blend_icon(uni_icon('icons/mob/species/diona/bodyparts.dmi', "diona_[body_part]", dir = SOUTH), ICON_OVERLAY)
+
+	for (var/markings_name in GLOB.diona_leaves_list)
+		var/datum/sprite_accessory/markings = GLOB.diona_leaves_list[markings_name]
+		var/datum/universal_icon/icon_with_markings = diona_body.copy()
+
+		if (markings_name != "None")
```

Comment:
define it or check existing defines

Reply:
Bellow too

Reply:
I am unsure what you mean by this.

Reply:
Define the string "None" instead of using it as plain string